Ginger/GingerCoreNET/Reports/HTMLReportConfiguration.cs

```diff
@@ -45,7 +45,8 @@ public  static partial class Fields
             public static string BusinessFlowFieldsToSelect = "BusinessFlowFieldsToSelect";
             public static string GingerRunnerFieldsToSelect = "GingerRunnerFieldsToSelect";
             public static string RunSetFieldsToSelect = "RunSetFieldsToSelect";
-            public static string ReportLowerLevelToShow = "ReportLowerLevelToShow";            
+            public static string ReportLowerLevelToShow = "ReportLowerLevelToShow";
+            public static string ExecutionStatisticsCountBy = "ExecutionStatisticsCountBy";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'ExecutionStatisticsCountBy' or make it 'const' or 'readonly'.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)